pyvista/plotting/utilities/sphinx_gallery.py

```diff
@@ -53,6 +53,14 @@ def html_rst(
     return images_rst
 
 
+def _process_events_before_scraping(plotter):
+    """Process events such as changing the camera or object before scraping."""
+    if plotter.iren is not None and plotter.iren.initialized:
+        if hasattr(plotter, "app") and plotter.app is not None:
```

Comment:
Nitpicky, but would you please add a detailed inline comment here about how `app` is specific to pyvistaqt and this is specific to handling that type of plotter (this will help us in the future)